src/main/java/Menu.java

```diff
@@ -0,0 +1,37 @@
+class Menu {
+    public static void main(String[] args) {
+        Drink drink1 = new Drink();
+        drink1.name ="Mojito";
```

Comment:
Checkstyle zauważył w tej linii błąd. To, co znalazł możesz podejrzeć w pull requeście z zakładce "Files changed". Polecam zainstalować sobie dodatek do intellij, żeby na bieżąco informował. Instrukcję znajdziesz tutaj https://bootcamp.javastart.pl/strona/jakosc-kodu